generators/entity-server/templates/src/main/java/package/service/dto/EntityDTO.java.ejs

```diff
@@ -54,7 +54,7 @@ import java.math.BigDecimal;
 <%_ if (fieldsContainBlob && databaseType === 'cassandra') { _%>
 import java.nio.ByteBuffer;
 <%_ } _%>
-<%_ if (fieldsContainOwnerManyToMany) { _%>
+<%_ if (fieldsContainOwnerManyToMany || fieldsContainEmbedded) { _%>
```

Comment:
What does `fieldsContainEmbedded` mean? That at least one of the fields is an embedded entity? Or all of them are?

Reply:
It's the same meaning of `fieldsContainOwnerManyToMany`, it means that the relationship contains at least one `embedded` entity